### ISSUE TITLE
File upload option doesn't appear after document creation

### ISSUE THREAD

**Issue by boikle** [GCRC/nunaliit#496]
**Issue/Bug:** When a schema definition defines a schema as having an attribute type of file, with the uploadOptional key set to a value of true. The ability to upload a file at a later time is not available to the user. 

**Suggestion:** If a user doesn't provide a file during the document's creation, allow the File: browse button field to be available at a later date. This ability may be a useful feature for people wishing to replace an attached file to a document, as well.

**Comment by jpfiset** [comment]
This mostly have to do with workflow. You should talk to Amos about it.

To turn on attachment editing, in nunaliit_custom:

````js
		customService.setOption('editorEnableAddFile',true);

````

**Comment by ahayes** [comment]
I agree that when a schema definition defines a file attribute, and the document being edited doesn't have a file, we should show the add file button. Likewise, when it does have a file, we should show the file and remove UI. These two things would help with file management yet still help avoid multiple file attachments without the customService mentioned by @jpfiset.
